crates/miden-lib/asm/kernels/transaction/lib/prologue.masm

```diff
@@ -444,6 +445,11 @@ proc.process_account_data
     exec.account::save_account_procedure_data
     # => [ACCOUNT_COMMITMENT]
 
+    # duplicate the initial storage slots in memory to enable a diff computation
+    # for the account delta
+    exec.memory::mem_copy_initial_storage_slots
```

Comment:
nit: since this procedure explicitly deals with only native account, should we name it something like `mem_copy_native_account_initial_storage_slots`?